### DISCUSSION TITLE
Why redact sent emails?

### DISCUSSION THREAD

**Discussion by jwclack52** [forwardemail/forwardemail.net#236] [General]
Just wondering why sent emails are redacted after 0-30 days.

**Comment by titanism** [comment]
Hi there 👋   Sent emails are actually redacted instantly by default and the body/contents of the message are replaced with this text:

```txt
This message was successfully sent. It has been redacted and
purged for your security and privacy. If you would like to
increase your message retention time, please go to the
Advanced Settings page for your domain.
```

You can configure the duration, note that if you use IMAP, then a cold copy will be stored in the Sent Mail folder.  IMAP storage is encrypted SQLite using your password which only you have access to (it's all done in-memory).

We do this to protect your security and privacy of messages/content -- that's the reason why.

**Comment by jwclack52** [comment]
That's (IMAP copy) great to hear!  I think you should clarify that in your docs and resources - it's a selling point!
